ci/cloudbuild/build.sh

```diff
@@ -351,6 +351,11 @@ if [[ "${DOCKER_FLAG}" = "true" ]]; then
     # generator dir.
     "--env=GENERATE_GOLDEN_ONLY=${GENERATE_GOLDEN_ONLY-}"
   )
+  if [[ -r "$HOME/.cloudcxxrc" ]]; then
```

Comment:
Nit: If you're going to put braces around `HOME` everywhere else, you might as well do it here too.

Reply:
Done.